<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-8CAR6C7" failures="1" tests="1" name="runners.NegativeAmountRunner" time="13.117" errors="0" timestamp="2025-03-18T12:32:08 IST" skipped="0">
  <testcase classname="runners.NegativeAmountRunner" name="runScenario" time="13.117">
    <failure message="Error message mismatch! expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Error message mismatch! expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at stepDefinitions.NegativeAmountSteps.user_should_see_the_error_message_negative_amount(NegativeAmountSteps.java:77)
at âœ½.User should see the error message "Invalid amount entered" for negative amount transfer(file:///C:/Users/dpraj/Downloads/Zero_Bank/Zero_Bank/Zero_Capston_Project/src/test/resources/features/NegativeAmount.feature:11)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
</testsuite> <!-- runners.NegativeAmountRunner -->
